source/_integrations/reolink.markdown

```diff
@@ -453,6 +453,8 @@ Removing a chime from a doorbell can be done by deleting the chime following the
 3. Select the chime you want to remove from the list of devices
 4. Underneath the **Device info**, select the three-dot {% icon "mdi:dots-vertical" %} menu. Then, select **Delete**.
 
+This will also decouple the chime from the doorbell in the reolink app/client. Therefore the chime will not ring anymore when the doorbell is pressed.
```

Comment:
```suggestion
This will also decouple the chime from the doorbell in the Reolink app/client. Therefore the chime will not ring anymore when the doorbell is pressed.
```